### ISSUE TITLE
Non-RO parts that RP-1 costs/places

### ISSUE THREAD

**Issue by NathanKell** [KSP-RO/RP-0#1502]
liquidEngineMini_v2
liquidEngine2-2
OrbitalScanner
SurfaceScanner
SurveyScanner
ServiceBay_125
ServiceBay_250
SXTelevonSmall
SXTelevonSmallHalf
SXTelevonLarge
SXTelevonVeryLarge
SXTWingSmall
SXTNK12M
ParaDockingPort
mk1pod
mk2LanderCabin

We should either not place/price them, or we should support them in RO.
I presume some of these are due to stock KSP changes (the old mk1pod, e.g.)

**Comment by DRVeyl** [comment]
Huh.
RO duplicates the liquidEngineMini_v2 into the RO-LMAE, but doesn't do anything with the original part after.  It doesn't do anything with the liquidEngineMini (non-v2).  I don't believe anything was lost in the big cleanup of the Squad engine patches.

Similar is true of both forms of liquidEngine2-2 and _v2.

Note RO's Squad engine handling deletes a handful of parts at the very end: <https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg#L1285-L1297>  but never deleted those, although it doesn't configure them.

I'm of the mindset that those 2 should be deleted in that segment 1285-1297.  RO's set a precedent for copying them for model purposes into whatever it wants, which is good, as that tends to be more clear about what "real" counterpart is rather than remembering or commenting which thing gets to be the "stock" part.

You're correct about the mk1Pod and mk2LanderCabin: RO is handling only the _v2 versions of those.  Should we do the same (delete the part) with the original versions that no longer have any RO patching?  Does that seem better from a long-term maintenance standpoint than adding them in with all the same patches that reference the _v2?

**Comment by NathanKell** [comment]
Yeah, we should probably do the same--I don't think we should _ever_ be in the business of patching parts that RO doesn't.

**Comment by NathanKell** [comment]
(or rather, that RO doesn't support, even if that support is just adding `RSSROConfig`)

**Comment by DRVeyl** [comment]
Sorry, when I asked "Should we do the same (delete the part) with the original versions that no longer have any RO patching? Does that seem better from a long-term maintenance standpoint than adding them in with all the same patches that reference the _v2?" I was actually asking from the perspective of RO, not from RP-1.

To clarify: Should *RO* delete the part, given that it is intentionally not applying any RO patches to it?

**Comment by NathanKell** [comment]
Oh. Yeah, probably. If it is intentionally not applying patches, but is cloning and applying patches, makes sense. (And same for patching the v2 versions but leaving v1s around)
